docs/zh/latest/FAQ.md

```diff
@@ -486,3 +486,17 @@ apisix:
 重启或 reload APISIX，所有 IP 便可以访问 `Admin API`。
 
 **注意：您可以在非生产环境中使用此方法，以允许所有客户端从任何地方访问您的 `Apache APISIX` 实例，但是在生产环境中使用它并不安全。在生产环境中，请仅授权特定的 IP 地址或地址范围访问您的实例。**
+
+## 基于acme.sh自动更新apisix ssl证书
+
+```bash
+$ curl --output /root/.acme.sh/renew-hook-update-apisix.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-apisix.sh
+
+$ chmod +x /root/.acme.sh/renew-hook-update-apisix.sh
+
+$ acme.sh  --issue  --staging  -d demo.domain --renew-hook "~/.acme.sh/renew-hook-update-apisix.sh  -h http://apisix-admin:port -p /root/.acme.sh/demo.domain/demo.domain.cer -k /root/.acme.sh/demo.domain/demo.domain.key -a xxxxxxxxxxxxx"
```

Comment:
I think it would be more suitable to be a blog instead of a FAQ answer?
This feature is not part of APISIX.

We can refer to the blog in this section:
https://github.com/apache/apisix#videos-and-articles

Reply:
https://github.com/apache/apisix/issues/3841#issuecomment-844628829

Reply:
I see. Could you provide an English version via an online translator like Youdao?

Reply:
Translate my blog or this PR?

Reply:
@spacewander  added zh and en doc